### PR TITLE
Story/bods 2286

### PR DESCRIPTION
**Pull request by alvak1**
Changes to load operating date exceptions as listed below

https://kpmgengineering.atlassian.net/browse/BODS-2286

### CODE REVIEW

File: transit_odp/common/xmlelements/elements.py [hunk 1]
```diff
@@ -190,3 +190,8 @@ def parent(self):
     def line_number(self):
         """Returns the line number of the element."""
         return self._element.sourceline
+
+    @property
+    def tag_localname(self):
```
Comment (pritish-ranjan01) [inline transit_odp/common/xmlelements/elements.py:195]: Can we reuse the existing localname function?

File: transit_odp/timetables/dataframes.py [hunk 2]
```diff
@@ -302,19 +305,138 @@ def flexible_operation_period_to_dataframe(flexible_vechicle_journeys):
...
+    else:
+        operating_profile_list.append(
+            get_operating_profile_with_exception(operating_profile)
+        )
+
+    return operating_profile_list
+
+
+def populate_operating_profiles(
+    operating_profile_services, vehicle_journey_code, service_ref
```
Comment (pritish-ranjan01) [inline transit_odp/timetables/dataframes.py:400]: The operating profiles could be from vehicle journeys as well. So can we keep the name generic as operating_profiles?

Comment (alvak1) [reply transit_odp/timetables/dataframes.py]: done

File: transit_odp/pipelines/tests/test_dataset_etl/data/test_operating_dates_exception/test_op_dates_exception_services_only.xml
[the remark this answers is not in the document]
Comment (alvak1) [reply transit_odp/pipelines/tests/test_dataset_etl/data/test_operating_dates_exception/test_op_dates_exception_services_only.xml:1]: done

File: transit_odp/common/xmlelements/elements.py (hunk 1 above)
Comment (alvak1) [reply transit_odp/common/xmlelements/elements.py]: done

**Review by pritish-ranjan01 — APPROVED** [review]

**Pull request merged**